recipes/aaf/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class AafConan(ConanFile):
+    name = "aaf"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://sourceforge.net/projects/aaf/"
+    description = "A  cross-platform SDK for AAF. AAF is a metadata management system and file format for use in professional multimedia creation and authoring."
+    topics = ("aaf", "multimedia", "crossplatform")
+    license = "AAF SDK PUBLIC SOURCE LICENSE"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+            "shared": [True, False],
+            "fPIC": [True, False]
+            }
+    default_options = {
+            "shared": False,
+            "fPIC": True
+            }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler == "Visual Studio" and self.settings.compiler.runtime in ("MT", "MTd"):
+            raise ConanInvalidConfiguration("Static runtime not supported")
+        if self.settings.os == "Linux":
+            raise ConanInvalidConfiguration("Linux not supported yet")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+        cmake = CMake(self)
+
+        if tools.is_apple_os(self.settings.os):
+            cmake.definitions["PLATFORM"] = "apple-clang"
+        elif self.settings.compiler == "Visual Studio":
+            cmake.definitions["PLATFORM"] = "vc"
+
+        cmake.definitions["ARCH"] = "x86_64"
```

Comment:
```suggestion
        cmake.definitions["ARCH"] = "x86_64"
```
Reading the cmake files, it works for folder name only.

Reply:
Indeed. Regarding your suggesion, I cannot commit it:
![Capture d’écran 2021-07-22 à 17 16 21](https://user-images.githubusercontent.com/879889/126663935-765499d5-acd5-48c4-bba3-b45d62cedf7a.png)

Reply:
there is no change in diff? or I just can't see it? @uilianries

Reply:
It's not a suggestion, I marked the diff to show about which line I'm commenting. My point is: It's fine using that definition, because is about folder name only, not arch support.